orchagent/intfsorch.cpp

```diff
@@ -382,6 +382,7 @@ void IntfsOrch::doTask(Consumer &consumer)
     while (it != consumer.m_toSync.end())
     {
         KeyOpFieldsValuesTuple t = it->second;
+        MacAddress mac;
```

Comment:
This can be moved to lines 408 - 410, where variables used for the same purpose are declared.

Reply:
good point, done